df-valve/src/main/java/com/hashmapinc/tempus/witsml/valve/dot/DotAuth.java

```diff
@@ -25,27 +25,41 @@
 import com.mashape.unirest.http.exceptions.UnirestException;
```

Comment:
Moved API_KEY to instance variable, made fields FINAL, added some nice commenting for getJWT